### DISCUSSION TITLE
How to reset email verification

### DISCUSSION THREAD

**Discussion by elsenchai** [xulihang/ImageTrans-docs#510] [Q&A] [answered]
Dumb question I know.
How do I reset my verification key when I'm changing PC? I tried finding the reset button in my old PC, and I don't see it in the documentations.

Thanks

**Comment by xulihang** [comment]
You can run ImageTrans on three devices. If you are trying to use it on a fourth device, it will show you a form to reset the device list.

**Comment by elsenchai** [comment]
My issue right now is when I enter my credentials it says Not Purchased

Is there any log files I can export for you to have a look?

Or is my license does not match the software version? Im using the latest ver. from OneDrive

**Comment by xulihang** [comment, marked as answer]
Free upgrades are within 6 months since your purchase.

You need to purchase it again to use new versions.
